custom_typings/cleankill.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module 'cleankill' {
+  interface InterruptHandler {
+    (done: (message?: string) => void): void;
```

Comment:
I haven't even gotten the typescriptified release of cleankill up on npm yet (blocked on https://github.com/PolymerLabs/cleankill/issues/4). After that I can delete this, and then eventually I can update to the version with promises.